observability/otel-bridge/examples/fastify-basic/README.md

```
@@ -0,0 +1,131 @@
+When you send a request with trace context, the server logs should show:
+
+```
+[OtelBridge] Extracted context from active span [traceId=4bf92f3577b34da6a3ce929d0e0e4736]
+```
+
+The Mastra spans will use the provided traceId and parentSpanId, maintaining trace continuity.
+
+## Key Implementation Details
+
+- **Import Order**: `instrumentation.ts` MUST be imported first
+- **No Plugin**: OTEL auto-instrumentation handles everything
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)